### ISSUE TITLE
Dragon monster overhaul

### ISSUE THREAD

**Issue by draconisPW** [draconisPW/PWMAngband#524]
Original balance towards spacing the different dragons from depth 1 to 100 didn't get well as shown by absurd endgame forms for mimics, dragon race and overall game difficulty due to ancient wyrms (not mentioning those can summon up to 8 copies of themselves endlessly). This needs to be rebalanced.

Endgame forms:

name:gold Thunderlord
speed:130
hit-points:4095
depth:70

name:Maia of Varda
speed:130
hit-points:5000
depth:71

name:greater Balrog
speed:130
hit-points:4500
depth:85

The ancient wyrms should be on par with these. Note that:

name:Great Wyrm of Power
speed:120
hit-points:5200
depth:84

is fine, as they have roughly the same HPs and less speed. What we should do for ancient wyrms: reduce their speed (125 seems a good value) and HP + damage output.

Need also to look back at V values for dragons, which were severly inflated probably too hastily. A good example is the sky dragon vs the sky drake.

**Comment by draconisPW** [comment]
After rebalancing from V:
- baby dragons slightly nerfed (88-120 hps instead of 88-132)
- young dragons buffed quite a bit (237-292 hps instead of 176-264)
- mature/ancient dragons and great wyrms untouched

Sky drake: lost 10 speed and 400 hps

Ancient wyrms: lost 5 speed + one dice on last two attacks, hp range 5500-6500 instead of 6500-8000

**Comment by draconisPW** [comment]
While comparing sorceror with full mana and mimics with endgame forms, I found out I overnerfed sorceror mana pool due to a bug in accounting for mana on items and capping the total mana pool vs +10 instead of the items. Original pool was 1000 (400 base mana, 25% flat bonus then +100% from items), new pool was 800 (400 + 100% from everything). To keep the same max value (since all bonuses are now additive and not multiplicative), I need to raise the final cap to +15 while keeping a +10 cap on items.
